ci-operator/config/code-ready/crc/code-ready-crc-master.yaml

```diff
@@ -20,9 +20,6 @@ images:
     openshift_release_golang-1.13:
       as:
       - registry.svc.ci.openshift.org/openshift/release:golang-1.13
-    root:
-      as:
-      - builder
```

Comment:
Was this someone thinking they could use it to replace the `as builder` instead of the literal pullspec, or is this a replacement feature we didn't know about?

Reply:
So this is because it replaces an alias to something that is already being replaced in which case it will not do the replacement: https://github.com/openshift/builder/blob/1205194b1d67f2b68c163add5ae17e4b81962ec3/pkg/build/builder/common.go#L489